src/TS3Connection.ts

```diff
@@ -457,23 +459,24 @@ export class TS3Listener extends events.EventEmitter {
      * - a mapping of the TS-UID to the Discord-username is created
      */
     private async tagUp(g: discord.Guild, commander: Commander) {
-        log("info", `Tagging up ${(commander.getTS3DisplayName())} in ${g.name}.`);
+        LOG.info(`Tagging up ${(commander.getTS3DisplayName())} in ${g.name}.`);
         const registration = this.botgartClient.registrationRepository.getUserByAccountName(commander.getAccountName());
 
         let duser: discord.GuildMember | undefined;
         if (registration !== undefined) {
             // the commander is member of the current discord -> give role
             duser = await g.members.fetch(registration.user); // cache.find(m => m.id === registration.user);
             if (duser === undefined) {
-                log("warning", `Tried to find GuildMember for user with registration ID ${registration.user}, but could not find any. Maybe this is a caching problem?`)
+                LOG.warn(`Tried to find GuildMember for user with registration ID ${registration.user}, but could not find any. Maybe this is a caching problem?`)
             }
             commander.setDiscordMember(duser);
 
             await this.tagUpAssignRole(g, commander);
         }
 
-        const message = await this.botgartClient.tagBroadcastService.sendTagUpBroadcast(g, commander, duser, registration);
-        commander.setBroadcastMessage(message)
+        await this.botgartClient.tagBroadcastService.sendTagUpBroadcast(g, commander, duser, registration)
```

Comment:
Does it make sense to use `await` in combination with `then` in this instance? Aka can `sendTagUpBroadcast` produce a `Promise<Promise<?>>`? If not, I would recommend consistent usage of `try-catch` instead, as is done in https://github.com/FlussuferOrga/Botgart/pull/146/files#diff-e5eeab1826518f1903f67129d4a4c1717555daffc84f1fa6de08197c7d96a2cfR497

Reply:
A Promise resolving to a promise ? Why ? 
Also this is much shorter than a Try catch.

Reply:
Sorry, I either clicked the wrong line or imagined things...
I thought I saw 

```javascript
await f().then(x => g(x))
```
somewhere, which would be the same as

```javascript
const x = await f();
g(x);
```

but would make a difference if `x` was a Promise itself.